src/devhub/devhub.js

```diff
@@ -142,7 +142,7 @@ async function mainSeeds() {
           <td><code>${record.command}</code></td>
           <td><time>${seedDuration}</time></td>
           <td><time>${seedFreshness} ago</time></td>
-          <td>${record.count}</td>
+          <td>${record.count.toLocaleString('en-US').replace(/,/g, ' ')}</td>
```

Comment:
```suggestion
          <td>${record.count.toLocaleString('en-US').replace(/,/g, '&nbsp;')}</td>
```

we do want this to be non-breaking, otherwise the same number will be split over multiple lines.

Reply:
Oops, that didn't occur to me!

Reply:
It didn't occur to me either, it's just that when I was manually prepping the screenshots, I got the wrapping with plain spaces!